mkt/extensions/migrations/0009_auto_20150921_0817.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we rename to something descriptive? I really don't like the `auto_timestamp` thing.